integration/actions.go

```diff
@@ -90,6 +90,17 @@ func Chain(actions ...Action) Action {
 	}
 }
 
+// SubTests runs a list of individual actions as separate tests
+func SubTests(tests ...SubTest) Action {
+	return func(t testing.TB, ic TestContext) {
```

Comment:
@alecthomas are you ok with this approach for running multiple assertions against a single instance?

Reply:
`Chain()` already does functionally the same thing, though this uses named subtests so is slightly different. Seems reasonable.